lib/rules/color-named/index.js

```diff
@@ -49,7 +50,7 @@ const rule = function (expectation, options) {
     const namedColors = Object.keys(namedColorData)
 
     root.walkDecls(decl => {
-      if (propertySets.acceptCustomIdents.has(decl.prop)) {
+      if (propertySets.acceptCustomIdents.has(decl.prop) || !isStandardSyntaxDeclaration(decl)) {
```

Comment:
use `isStandardSyntaxDeclaration` we disabled check on color in variables, root and scss nested properties (see https://github.com/stylelint/stylelint/blob/master/lib/utils/isStandardSyntaxDeclaration.js).
@stylelint/core what do you think guys? I vote for the exclusion of non-standard syntax.